CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 9.0.0
```

Comment:
keep that as `Unreleased`, as craft will put the version there automatically ;-)

Reply:
Good to know, thanks!